packages/massa-proto-cli/src/ts-gen.ts

```diff
@@ -0,0 +1,18 @@
+import { ProtoFile } from './protobuf';
+
+function generateArgVerification(protoFile: ProtoFile): string {
+  let content = '';
+  for (let arg of protoFile.argFields) {
+    if (
+      arg.type === 'uint32' ||
+      arg.type === 'uint64' ||
+      arg.type === 'fixed32' ||
+      arg.type === 'fixed64'
```

Comment:
You can simplify/clarify your code by regrouping all protobu unsigned int in an array and check that this arg.type is not in this array.

Something like:
```ts
  const unsignedPBTypes = new Set(['uint32', 'uint64', 'fixed32', 'fixed64']);
  ...
    if (unsignedPBTypes.has(arg.type)) {
```